test/integration/generated/body-string/Source/StringOperation.swift

```diff
@@ -334,11 +338,12 @@ public final class StringOperation {
 
     /// Set string value empty ''
     /// - Parameters:
-
+    ///    - empty :
```

Comment:
Is there no text for the body param, or are we missing it because of the name switch?

Reply:
It is missing due to name switch.

Reply:
issue is fixed after upgrading autoerest testserver version.